blockchain/emulator.go

```diff
@@ -94,8 +94,14 @@ func (e *emulator) executeTransaction(
 	script string,
 	arguments []string,
 	authorizers []flowsdk.Address,
-) (*types.TransactionResult, *flowsdk.Transaction, error) {
-	tx := &flowsdk.Transaction{}
+) (result *types.TransactionResult, tx *flowsdk.Transaction, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = errors.New("Panic occurred during transaction execution")
```

Comment:
Why is this needed? It hides the details of the panic and panic shouldn't be treated as the error. So I wonder what is your thinking?

Reply:
We actually don't need this 👀

Reply:
Okay :D